src/Avalonia.Controls.ColorPicker/ColorPicker/ColorPicker.cs

```diff
@@ -1,16 +1,72 @@
-﻿namespace Avalonia.Controls
+﻿using Avalonia.Controls.Metadata;
+using Avalonia.Controls.Primitives;
+using Avalonia.Controls.Templates;
+
+namespace Avalonia.Controls
 {
     /// <summary>
     /// Presents a color for user editing using a spectrum, palette and component sliders within a drop down.
     /// Editing is available when the drop down flyout is opened; otherwise, only the preview color is shown.
     /// </summary>
+    [PseudoClasses(PC_Empty)]
     public class ColorPicker : ColorView
     {
+        private const string PC_Empty = ":empty";
         /// <summary>
         /// Initializes a new instance of the <see cref="ColorPicker"/> class.
         /// </summary>
         public ColorPicker() : base()
         {
         }
+
+        /// <summary>
+        /// Defines the <see cref="Content"/> property. 
+        /// </summary>
+        public static readonly StyledProperty<object?> ContentProperty = AvaloniaProperty.Register<ColorPicker, object?>(
+            nameof(Content));
+
+        /// <summary>
+        /// Get or set the Content displayed in Dropdown Button. 
+        /// </summary>
+        public object? Content
+        {
+            get => GetValue(ContentProperty);
+            set => SetValue(ContentProperty, value);
+        }
+
+        /// <summary>
+        /// Defines the <see cref="ContentTemplate"/> property. 
+        /// </summary>
+        public static readonly StyledProperty<IDataTemplate?> ContentTemplateProperty = AvaloniaProperty.Register<ColorPicker, IDataTemplate?>(
+            nameof(ContentTemplate));
+
+        /// <summary>
+        /// Get or set the ContentTemplate displayed in Dropdown Button. 
+        /// </summary>
```

Comment:
> Gets or sets the data template used to display the content of the control.

https://github.com/AvaloniaUI/Avalonia/blob/7479f53750295f55dd637192e8df0539aec260ae/src/Avalonia.Controls/ContentControl.cs#L54-L61

I think just copy/pasting the docs here is best.